kalite/testing/base_environment.py

```diff
@@ -11,20 +11,19 @@
 from urlparse import urljoin
 from django.contrib.auth.models import User
 
-from kalite.testing.behave_helpers import login_as_admin, login_as_coach, logout
+from kalite.testing.behave_helpers import login_as_admin, login_as_coach, logout, login_as_learner
 
 def before_all(context):
-    browser = context.browser = webdriver.Firefox()
-    # ensure the window is reasonably sized.
-    browser.set_window_size(1024, 768)
+    pass
 
 def after_all(context):
-    try:
-        context.browser.quit()
-    except CannotSendRequest:
-        pass
+    pass
 
 def before_feature(context, feature):
+    browser = context.browser = webdriver.Firefox()
+    # ensure the window is reasonably sized.
+    browser.set_window_size(1024, 768)
```

Comment:
Trying adding extra wait time, as, still, it is passing perfectly on mine. Really quite frustrating.